_posts/2020-05-29-issue-3.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: 'Issue 3: An Invitation to Agonize'
+category: Volume 2
+image:
+    file: vol-2-issue-3-header.png
+    alt: 'Kim Tran Ph.D'
+    caption: null
+    source_link: null
+    half_width: false
+preview: 'Kim Tran shares her story of being born and raised in the Bay Area, doing diversity work in tech, and trying to make sense of what it means.'
+---
+
+In this issue, Kim Tran shares her story of being born and raised in the Bay Area, doing diversity work in tech, and trying to make sense of what it means.
```

Comment:
```suggestion
In this issue, [Kim Tran](https://twitter.com/but_im_kim_tran) shares her story of being born and raised in the Bay Area, doing diversity work in tech, and trying to make sense of what it means.
```

Should we link to Kim's twitter? (To give a bit more context on who she is)

Reply:
If Kim's fine with that, I'm fine with it. (I asked for her consent in the Google Doc, since she doesn't have a GitHub account and is not on this review thread.)